classes/ActionScheduler_Store.php

```diff
@@ -150,6 +150,49 @@ protected function validate_sql_comparator( $comparison_operator ) {
 		return '=';
 	}
 
+	/**
+	 * Get the time MySQL formated date/time string for an action's (next) scheduled date.
+	 *
+	 * @param ActionScheduler_Action $action
+	 * @param DateTime $scheduled_date (optional)
+	 * @return string
+	 */
+	protected function get_scheduled_date_string( ActionScheduler_Action $action, DateTime $scheduled_date = NULL ) {
+		$next = is_null( $date ) ? $action->get_schedule()->next() : $date;
```

Comment:
These instances of `$date` should instead be `$scheduled_date`.

From a performance perspective, it would be preferable to use `null === $scheduled_date` instead of `is_null( $schdeuled_date )`. The reason is that a function call is a bit slower than the explicit comparison (example here: https://stackoverflow.com/a/4662691). With that said, it's also worth noting that the difference seems to much less, or even nonexistent, as of PHP7 (see the replies to the previous link).

Reply:
I'll wait to see what happens with https://github.com/Prospress/action-scheduler-custom-tables/pull/26/ before addressing the `$date` vs `$scheduled_date` issue here as it will likely require/benefit from more substantial refactoring if that is merged, like abstracting part of into new `get_date_string()`/`get_date_string_local()` methods.